ui/js/controllers/perf/graphs.js

```diff
@@ -58,6 +59,15 @@ perf.controller('GraphsCtrl', [
             });
         };
 
+        $scope.nudgeAlert = (dataPoint, direction) => {
+            $scope.nudgingAlert = true;
+
+            const resultSetData = dataPoint.series.flotSeries.resultSetData;
+            PhAlerts.nudgeAlert(dataPoint, direction, resultSetData);
+
+            $scope.nudgingAlert = false;
```

Comment:
This won't work as the nudgeAlert function works asynchrously. You'll want to set/unset the nudgingAlert variable inside the promise itself, I think